# SteeringAngles.msg - 四轮角度反馈
# 转向角度信息
float64 steering_01     # 1号转向轮角度 (单位: rad)
float64 steering_02     # 2号转向轮角度 (单位: rad)
float64 steering_03     # 3号转向轮角度 (单位: rad)
float64 steering_04     # 4号转向轮角度 (单位: rad)